lms/views/predicates/_lti_launch.py

```diff
@@ -41,13 +41,21 @@ def __call__(self, context, request):
         resource_link_id = request.params.get("resource_link_id")
         tool_consumer_instance_guid = request.params.get("tool_consumer_instance_guid")
 
-        has_module_item_configuration = (
-            ModuleItemConfiguration.get_document_url(
-                request.db, tool_consumer_instance_guid, resource_link_id
-            )
-            is not None
+        document_url = ModuleItemConfiguration.get_document_url(
+            request.db, tool_consumer_instance_guid, resource_link_id
         )
 
+        if document_url:
+            has_module_item_configuration = True
+        else:
```

Comment:
We really need to document the hell out of the fact this is Blackboard specific. Especially as this is undocumented, there's just no handle to grab onto in future to work out what this does and why.